src/render_engine_cli/utils.py

```diff
@@ -212,3 +212,20 @@ def get_editor(ctx: click.Context, param: click.Option, value: str) -> str | Non
             return None
         case _:
             return value
+
+
+def handle_content_file(ctx: click.Context, param: click.Option, value: str) -> str | None:
+    """Handle the content file"""
+    if value == "stdin":
+        content = list()
+        click.secho('Please enter the content. To finish, put a "." on a blank line.', fg="green")
+        while (line := input("")) != ".":
+            content.append(line)
+        print(content)
+        return "\n".join(content)
+    path = Path(value)
+    if not path.exists:
```

Comment:
```suggestion
    if not path.exists():
```

Reply:
exist is a [function](https://docs.python.org/3/library/pathlib.html#pathlib.Path.exists) not a property